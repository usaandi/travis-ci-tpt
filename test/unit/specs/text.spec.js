function text(t = '') {
  return t;
}

// test case1: returns same text as given in param
// test case2: if no text given, returns empty string
describe('text', () => {
  it('returns same text as given in param', () => {
    const t1 = text('kala');
    expect(t1).toBe('kala');
    expect(t1).toMatchSnapshot();
  });
  it('if no text given, returns empty string', () => {
    const t1 = text();
    expect(t1).toBe('');
  });

  it('if no text given, returns empty string', () => {
    const t1 = text();
    expect(t1).toBe('');
  });
});
