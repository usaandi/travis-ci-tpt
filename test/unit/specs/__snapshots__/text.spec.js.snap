// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`text returns same text as given in param 1`] = `"kala"`;
